### PR TITLE
Add the omnisharp 1.32.13 updates

### PR DESCRIPTION
**Pull request by akshita31**

### CODE REVIEW

**Comment by akshita31** [comment]
PS: We need to change the date in the changelog and update the version in the readme.md and change the link in the readme.md

File: src/observers/TelemetryObserver.ts [hunk 1]
```diff
@@ -47,9 +47,9 @@ export class TelemetryObserver {
             case EventType.ProjectConfigurationReceived:
                 let projectConfig = (<ProjectConfiguration>event).projectConfiguration;
                 telemetryProps['ProjectGuid'] = projectConfig.ProjectGuid;
```
Comment (NTaylorMullen) [inline src/observers/TelemetryObserver.ts]: I believe you should also change this to `ProjectId`. Right @DamianEdwards?

Comment (akshita31) [reply src/observers/TelemetryObserver.ts]: Done. Thanks!

Comment (Andrew-MSFT) [reply src/observers/TelemetryObserver.ts]: Perfect :-)

**Comment by akshita31** [comment]
If https://github.com/OmniSharp/omnisharp-vscode/pull/2873 is merged by Monday, we can merge it and do the release for 1.18.1.

**Pull request merged**